docs/book/installation/installation.rst

```diff
@@ -89,29 +82,16 @@ And now you can use gulp for installing views, by just running a simple command:
 Accessing the Shop
 ------------------
 
-.. tip::
-
-    We strongly recommend using the Symfony built-in web server by running the
-    ``php bin/console server:start``
-    command and then accessing ``http://127.0.0.1:8000`` in your web browser to see the shop.
+We strongly recommend using the Symfony built-in web server by running the ``php bin/console server:start``
+command and then accessing ``http://127.0.0.1:8000`` in your web browser to see the shop.
 
 .. note::
 
     The localhost's 8000 port may be already occupied by some other process.
-    If so you should try other ports, like for instance:
-    ``php bin/console server:start 127.0.0.1:8081``
+    If that happens, please try using a different port - ``php bin/console server:start 127.0.0.1:8081``.
     Want to know more about using a built-in server, see `here <http://symfony.com/doc/current/cookbook/web_server/built_in.html>`_.
```

Comment:
```suggestion
    Get to know more about using a built-in server `here <http://symfony.com/doc/current/cookbook/web_server/built_in.html>`_.
```

Reply:
👍, can also improve the accessibility by creating a more meaningful link than "here".